atlassian_jwt_auth/frameworks/django/backend.py

```diff
@@ -0,0 +1,51 @@
+from django.conf import settings as django_settings
+from django.http import HttpResponse, HttpResponseForbidden
+from ..common.backend import Backend
+from ..common.utils import SettingsDict
+
+
+class DjangoBackend(Backend):
+    def get_authorization_header(self, request=None):
+        if request is None:
```

Comment:
If you're referring to the optional `request=None` argument above, I left that to maintain the exact same API as the base Backend.
Otherwise, that's essentially what I'm doing, right? I'm checking to see if `request is None` and if it is, I'm raising an exception.